dev/loadgen/prod-benchmark.yaml

```diff
@@ -1,7 +1,7 @@
 ## start with
 ##    loadgen benchmark prod-benchmark.yaml
 
-workspaces: 500
+workspaces: 60
```

Comment:
this is intentional? seems like it shouldn't be part of this PR?

Reply:
Yes, but I can change that in a different PR. Fixed